### PR TITLE
Some cleanup & typing improvements

### PR DESCRIPTION
**Pull request by jsangmeister**
Improvements done while investigating https://github.com/OpenSlides/openslides-backend/issues/1801

### CODE REVIEW

**Comment by jsangmeister** [comment]
@r-peschke ping

File: openslides_backend/action/actions/meeting/import_.py [hunk 1]
```diff
@@ -575,7 +575,7 @@ def create_events(
         )
 
         # add meetings to organization if set in meeting
-        adder: Dict[str, List[Union[int, str]]] = {}
+        adder: ListFieldsValue = {}
```
Comment (r-peschke) [inline openslides_backend/action/actions/meeting/import_.py:578]: Please revert this file. A type ListFieldsValue for a dict is the opposite of readable

**Review by r-peschke — CHANGES_REQUESTED** [review]
Just one thing

File: openslides_backend/action/actions/meeting/import_.py (hunk 1 above)
Comment (r-peschke) [reply openslides_backend/action/actions/meeting/import_.py:578]: I still think the best is to revert it.
I asked myself, why it is a union? One would expect the meeting_id to be always int.

**Review by r-peschke — CHANGES_REQUESTED** [review]

File: openslides_backend/action/actions/meeting/import_.py (hunk 1 above)
Comment (jsangmeister) [reply openslides_backend/action/actions/meeting/import_.py:578]: `Dict` is [invariant](https://mypy.readthedocs.io/en/stable/common_issues.html#variance) and the `list_fields` can potentially be used with integers and strings, therefore, the type used here must be a union as well. But this is actually an argument *for* a type alias: You see a union of strings and integers and are confused at first. If we just use some type alias, this confusion can be avoided. Additionally, this rather verbose type is used in 3 separate places, so it also goes against the DRY principle to repeat the type. As I said, I'm open for another name suggestion if `ListFieldsValue` does not suit you.

Comment (jsangmeister) [reply openslides_backend/action/actions/meeting/import_.py]: renamed to `ListFieldsDict`

**Review by r-peschke — APPROVED** [review]

**Pull request merged**